### PR TITLE
fix(inputbase): update width prop

### PR DESCRIPTION
**Pull request by katieweinstein**
Issue: #

## What I did
Added width prop to InputBase so that width is passed through to the input element.

## Checklist 
- [x] changeset has been added
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/react-magma-dom/src/components/InputBase/index.tsx [hunk 1]
```diff
@@ -114,6 +114,10 @@ export interface InputBaseProps
    * Boolean for whether this is a Password Input or not
    */
   isPasswordInput?: boolean;
+  /**
+   * String to determine width of input
+   */
+  width: string;
```
Comment (silvalaura) [inline packages/react-magma-dom/src/components/InputBase/index.tsx:120]: I think we want to make this prop optional:
```suggestion
  width?: string;
```

File: .changeset/fix-inputWrapper.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"react-magma-dom": patch
+---
+
+fix(inputbase): update width prop
```
Comment (silvalaura) [inline .changeset/fix-inputWrapper.md:5]: The changeset is what gets used to populate our [changelog](https://react-magma.cengage.com/version/3.2.1/api-intro/changelog/). The more descriptive we are here, the easier it is for adopters to understand what changes we introduce in each version.

**Review by silvalaura — COMMENTED** [review]
Looks like the build is breaking because of Passport Input. We may need to add the `width` prop to all inputs that use InputBase.

File: packages/react-magma-dom/src/components/InputBase/index.tsx [hunk 3]
```diff
@@ -114,6 +114,10 @@ export interface InputBaseProps
    * Boolean for whether this is a Password Input or not
    */
   isPasswordInput?: boolean;
+  /**
+   * String to determine width of input
```
Comment (silvalaura) [inline packages/react-magma-dom/src/components/InputBase/index.tsx:118]:
```suggestion
   * String to determine width of input, must be suffixed with "px", "rem", or "%""
```

File: packages/react-magma-dom/src/components/PasswordInput/index.tsx [hunk 4]
```diff
@@ -112,6 +117,8 @@ export const PasswordInput = React.forwardRef<
 
   const isInverse = useIsInverse(props.isInverse);
 
+  console.log(props.width);
+
```
Comment (silvalaura) [inline packages/react-magma-dom/src/components/PasswordInput/index.tsx:121]: We should remove this

File: .changeset/fix-inputWrapper.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': patch
+---
+
+fix(inputbase): Add and update width prop for InputBase
```
Comment (silvalaura) [inline .changeset/fix-inputWrapper.md:5]: InputBase is used under the hood for many components, but adopters don't know this. We should mention the specific components it impacts: Input, PasswordInput.

Does this need to also be applied to Search?

**Review by silvalaura — COMMENTED** [review]
Ideally, we shouldn't push the package-lock. I wonder if that caused the build to fail?

**Review by silvalaura — APPROVED** [review]

**Review by ccedrone — APPROVED** [review]

**Comment by hkrishna29** [comment]
The preview of the components is looking fine at 400 %

**Pull request merged**